### PR TITLE
Redo technologies block with grid

### LINKED ISSUE #112: [REFACT] redo technologies block using flexbox / grid + extract to partial
Image
Image
Image
Image

Extract to partial
Feel free to ues sane html and flexbox / grid

Would be nice if they doesn't depend on html markup position, so i can later in loop dump it like this.

**Use homepage icons (they are svg).**

```
foreach ($items as $item)

<div>$item->name</div>

endforeach
```

### PR DESCRIPTION
**Pull request by AucT**
Closes #112 

## Summary by CodeRabbit

- **New Features**
	- Introduced a new modular layout for displaying technology information using a partial template with SVG graphics.
	- Added a new CSS stylesheet for enhanced styling of technology sections.

- **Bug Fixes**
	- Streamlined the inclusion of CSS files across various pages for consistency.
	- Removed outdated technology sections to improve content management and page structure.

- **Documentation**
	- Updated the structure of technology-related content to improve maintainability and responsiveness.

### CODE REVIEW

**Comment by AucT** [comment]
ZQzhxf8I44

File: themes/beaver/assets/css/technologies.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+.fl-node-w8dym2q746pj>.fl-row-content-wrap {
+    padding-top: 0;
+}
+/*
+quick fix use-case page
+ */
+.fl-node-r5xy9wufkc2e>.fl-row-content-wrap {
+    padding-top: 0;
+}
```
Comment (pftg) [inline themes/beaver/assets/css/technologies.css:70]: We agreed to use  rscss , please update it

**Review by pftg — CHANGES_REQUESTED** [review]
Let’s have consistent code

File: data/technologies.yaml [hunk 2]
```diff
@@ -1,12 +1,12 @@
 - name: Ruby on Rails
-  image: /assets/img/technologies/rube-on-rails.png
+  svg: technologies/ruby
```
Comment (pftg) [inline data/technologies.yaml]: I’d name it icon not svg

File: postcss.config.js [hunk 3]
```diff
@@ -30,7 +30,7 @@ const purgecss = require("@fullhuman/postcss-purgecss")({
     ],
 
     greedy: [
-      /^swiper-/, /^is-/, /^has-/, /^js-/, /^fl-builder-content/, /^fl-col/, /^fl-node/,
+      /^swiper-/, /^is-/, /^has-/, /^js-/, /^fl-builder-content/, /^fl-col/, /^fl-node/, /^technologies/
```
Comment (pftg) [inline postcss.config.js]: But it’s used by default on server rendering and is not required to be posted here

File: themes/beaver/assets/css/technologies.css [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    }
+    .technologies__description {
+        margin-left: 200px;
+        margin-right: 200px;
+    }
+}
+
+
+/*
+quick fix services page
```
Comment (pftg) [inline themes/beaver/assets/css/technologies.css:59]: 👍

File: data/technologies.yaml (hunk 2 above)
Comment (AucT) [reply data/technologies.yaml]: там partial юзає svg, тому так.

File: postcss.config.js (hunk 3 above)
Comment (AucT) [reply postcss.config.js]: @pftg  без цього PostCSS вижерає стилі

File: themes/beaver/assets/css/technologies.css (hunk 1 above)
Comment (AucT) [reply themes/beaver/assets/css/technologies.css:70]: ok

File: data/technologies.yaml (hunk 2 above)
Comment (pftg) [reply data/technologies.yaml]: There is no problem to rename as I know, why we cannot improve naming?

File: postcss.config.js (hunk 3 above)
Comment (pftg) [reply postcss.config.js]: Are you sure? It should not, because then we do not use it in html.

**Comment by AucT** [comment]
converted to rscss

xkD80H0XBv
PbOUnUJ6GZ
KH91800p3R
tnb5p7nrxz

**Review by pftg — APPROVED** [review]

**Pull request merged**